### DISCUSSION TITLE
Wifi Signal strength in Wifi configuration page for multiple APs

### DISCUSSION THREAD

**Discussion by durandeux** [roleoroleo/yi-hack-Allwinner-v2#849] [Ideas]
First, big thank you as I use the firmware on 6 cameras around the house (all connected to Frigate and HA). I have multiple wifi access points in my house. And it is not always easy to pick the right one for each camera.

Currently in the wifi configuration window, it displays the list of all available networks, but it does not give any indication of the signal strength. All my APs have the same name, so I don't know which one to pick. If we could display the signal strength next to each, it would be fantastic (see pic below that illustrates the problem with multiple APs in the list). I am attaching how this is displayed in OpenWRT for example.

I hope the idea makes sense.
Thanks!

![Capture d’écran 2024-03-28 103819](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/57592953/6b5d5117-ffe7-4cd5-9327-56818e56dea2)
![Capture d’écran 2024-03-28 104213](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/57592953/c6aa5ca7-4e19-4c89-9658-6153102ee837)

**Comment by roleoroleo** [comment]
I understand your question but, the cam (natively) uses just SSID and password to connect to the wifi network (not BSSID).
So you can't force the cam to connect to a specific AP.

**Reply by durandeux** [reply]
ohhhhhhhhhh that makes sense, thank you.
